README.md

```diff
@@ -70,26 +69,35 @@ The **test manifest** describes the tests to be run. The test manifest file typi
 example.test:
 
 {
-    'stl_files': ['example.stl'],
-
-    'roles': [
-        { 'role': 'example::rReceiver',
-          'ipAddress': $ip,  # Must pass --manifest-args="ip='0.0.0.0'" to run.
-          'transportId': 'receiver-0',
-        },
-    ],
-
-    'test': ['example::rReceiver'],
+  'stl_files': [
+    'example_base.stl',
+    'example.stl',
+  ],
+
+  'roles': [  # Role information
+    { 'role': 'example::rReceiver',
+      'ipAddress': $ip,  # Must pass --manifest-args 'ip="0.0.0.0"'
+      'transportId': 'receiver-0',
+    },
+  ],
+
+  'constants': {
+    'example::kHelloWorld': $hello_world,  # Must pass --manifest-args \
+                                           # 'hello_world="Hello, world!"'
+  },
+
+  'test': ['example::rReceiver'],
 }
 
-$ python test_driver.py example/example.test --manifest-args=”ip=’0.0.0.0’”
+$ python test_driver.py example/example.test --manifest-args 'ip="0.0.0.0"' \
+      'hello_world="Hello, world!"'
 ```
 
 ### 3.2. stl_files
-The ‘stl_files’ field must be a list of strings naming the STL files that include the transitions to be tested. The filenames here should be relative to the manifest .test file. In the example above, [example.test](example/example.test) and [example.stl](example/example.stl) are in the same directory.
+The ‘stl_files’ field must be a list of strings naming the STL files that include the transitions to be tested. The filenames here should be relative to the manifest .test file. In the example above, [example.test](example/example.test) and [example_base.stl](example/example_base.stl) are in the same directory.
 
 ### 3.3. roles
-The ‘roles’ field must be list of dicts. Each dict element in the list describes one role. These are the roles described in the stl_files listed in the ‘stl_files’ field. In the above example, there [example.stl](example/example.stl) contains the following:
+The ‘roles’ field must be list of dicts. Each dict element in the list describes one role. These are the roles described in the stl_files listed in the ‘stl_files’ field. In the above example, there [example_base.stl](example/example_base.stl) contains the following:
```

Comment:
nit: I know this was the original wording, but I think the 'there' should get removed so it just reads: In the above example, example_base.stl contains the following